### PR TITLE
docs: updated doc-site to match schema 3.0.0 requirements

### PR DESCRIPTION
**Pull request by nayib-jose-gloria**
- #3156 

### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
- updated doc site to match schema 3.0.0 requirements / links / screenshots
- removed a couple of unused images
- fixed dev env doc minor inconsistency

### CODE REVIEW

**Review by ebezzi — APPROVED** [review]

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 1]
```diff
@@ -34,26 +34,28 @@ We need the following collection metadata (i.e. details associated with your pub
   - Publication/preprint DOI: can be added later
```
Comment (brianraymor) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:34]: When Max wrote this section, @jahilton was the primary contributor and reviewers. Jason - any comments on [frontend/doc-site/032__Contribute and Publish Data.mdx](https://github.com/chanzuckerberg/single-cell-data-portal/pull/3180/files#diff-3aab6608e02aeee4406a95606f0a84d5f03989f2ababdb4fb87416f1eb5f8c7e) ?

File: frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_2__Cell Type and Gene Ordering.mdx [hunk 2]
```diff
@@ -4,7 +4,7 @@ Cell types and genes shown in a dot plot can be arranged in different ways as de
 
 # Cell Type Ordering
```
Comment (brianraymor) [inline frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_2__Cell Type and Gene Ordering.mdx:5]: This link needs to be updated to match the pinned ontology in the schema?

### Algorithm

`1. Load the CL ontology, using either the cl.obo or cl.owl from the source file [here](https://github.com/obophenotype/cell-ontology/releases/tag/v2021-08-10).`

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 3]
```diff
@@ -34,26 +34,28 @@ We need the following collection metadata (i.e. details associated with your pub
...
   - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
 
-Datasets need the following information added to a single h5ad (AnnData 0.7) format file per CELLxGENE visualization:
+Datasets need the following information added to a single h5ad (AnnData 0.8) format file per CELLxGENE visualization:
 
 - **Dataset-level metadata in uns**:
-  - schema_version: `2.0.0`
+  - schema_version: `3.0.0`
   - title: title of the individual dataset visualization
```
Comment (brianraymor) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:41]: I'm not sure why the original used the word _visualization_. I think it should be struck.

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 4]
```diff
@@ -34,26 +34,28 @@ We need the following collection metadata (i.e. details associated with your pub
...
   - optional: batch_condition: list of obs fields that define that “batches” that a normalization or integration algorithm should be aware of
```
Comment (brianraymor) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:42]: In the original:

that define that "batches" that -> that define the "batches" or that define "batches" ?

File: frontend/doc-site/03__Download Published Data.mdx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Downloading Published Data on CZ CELLxGENE Discover
 
-Clicking the download button launches a dialog that enables a dataset to be downloaded in h5ad (AnnData v0.7) and rds (Seurat v3) formats. All datasets adhere to the CELLxGENE single cell annotated data schema. Datasets can either be downloaded via the browser by clicking the blue download button, or via the command line by pasting the provided curl command.
+Clicking the download button launches a dialog that enables a dataset to be downloaded in h5ad (AnnData v0.8) and rds (Seurat v3) formats. All datasets adhere to the CELLxGENE single cell annotated data schema. Datasets can either be downloaded via the browser by clicking the blue download button, or via the command line by pasting the provided curl command.
 
 1. Select the data set you wish to download.
```
Comment (brianraymor) [inline frontend/doc-site/03__Download Published Data.mdx:5]: The images are not rendering in the file preview, but I would expect that we need new screen captures for the download dialog which will display anndata 0.7 ...

https://cellxgene.cziscience.com/docs/03__Download%20Published%20Data

**Review by brianraymor — CHANGES_REQUESTED** [review]
Minor changes requested.

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 6]
```diff
@@ -34,26 +34,28 @@ We need the following collection metadata (i.e. details associated with your pub
   - Publication/preprint DOI: can be added later
   - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
 
-Datasets need the following information added to a single h5ad (AnnData 0.7) format file per CELLxGENE visualization:
+Datasets need the following information added to a single h5ad (AnnData 0.8) format file per CELLxGENE visualization:
```
Comment (brianraymor) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:37]: I would strike _per CELLxGENE visualization_ - the information is required by the schema for many feature requirements related to metadata ...

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 7]
```diff
@@ -34,26 +34,28 @@ We need the following collection metadata (i.e. details associated with your pub
...
+  - schema_version: `3.0.0`
+  - title: title of the individual dataset
+  - optional: batch_condition: list of obs fields that define “batches” that a normalization or integration algorithm should be aware of
 - **Data in .X and raw.X**:
   - raw counts are required
   - normalized counts are strongly recommended
   - raw counts should be in raw.X if normalized counts are in .X
   - if there is no normalized matrix, raw counts should be in .X
-- **Cell metadata in obs (the values MUST be the most specific specific term available from the specified ontology)**:
+- **Cell metadata in obs (for ontology term IDs, the values MUST be the most specific term available from the specified ontology)**:
```
Comment (brianraymor) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:48]: @jahilton - was this list created in some priority order or could it be sorted? It seems random. I sort in the schema.

**Comment by brianraymor** [comment]
@pablo-gar - you may want to take a look at the sc expression image change ...

**Comment by pablo-gar** [comment]
> @pablo-gar - you may want to take a look at the sc expression image change ...

Looks good, was this change just for dimensions?

**Comment by brianraymor** [comment]
> Looks good, was this change just for dimensions?

No - ethnicity is being renamed to self-reported ethnicity in 3.0.0.

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 7 above)
Comment (jahilton) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:48]: It is random. When I present this non-randomly, I use something like this logical order of high-to-low level biology to technical:

- organism
- donor_id
- dev_stage
- sex
- ethnicity
- disease
- tissue
- cell_type
- assay
- suspension_type

Comment (nayib-jose-gloria) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:48]: @brianraymor should we update this list to follow the ordering jason outlined above?

Comment (brianraymor) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:48]: I think that would best since Jason can reuse this information for _first contact_.

**Comment by brianraymor** [comment]
@nayib-jose-gloria - LGTM.

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 6 above)
Comment (brianraymor) [reply frontend/doc-site/032__Contribute and Publish Data.mdx]: @nayib-jose-gloria ? Pending editorial?

Comment (nayib-jose-gloria) [reply frontend/doc-site/032__Contribute and Publish Data.mdx]: https://github.com/chanzuckerberg/single-cell-data-portal/pull/3180/files#diff-3aab6608e02aeee4406a95606f0a84d5f03989f2ababdb4fb87416f1eb5f8c7e

**Pull request merged**